### ISSUE TITLE
[CT-3265] The docs generated for exposures should display owner

### ISSUE THREAD

**Issue by siljamardla** [dbt-labs/dbt-docs#470]
### Describe the feature
Because 'owner' is a required property of an exposure, I would expect this to be displayed in the documentation generated. In fact, all of the properties of an exposure should be displayed. Maturity is also missing.

### Describe alternatives you've considered
Using the meta parameters to add additional owner information that does get displayed in the header of the docs, but this is not DRY.

### Additional context
Basic functionality for exposures

### Who will this benefit?
Anyone who ever defines an exposure

### Are you interested in contributing this feature?
Sure

**Comment by jairus-m** [comment]
Hello, this seems to still be an issue.

**Comment by ndrewwm** [comment]
Would be super nice to have this section of the docs site be restored! 🙏 🙏

**Comment by astephan91** [comment]
I agree, I feel like this is a big miss from the documentation site.
